edx_rest_framework_extensions/authentication.py

```diff
@@ -148,3 +149,62 @@ def process_user_info_response(self, response):
 
     def authenticate_header(self, request):
         return 'Bearer'
+
+
+class JwtAuthentication(JSONWebTokenAuthentication):
+    """
+    JSON Web Token based authentication.
+
+    This authentication class is useful for authenticating a JWT using a secret key. This can be a shared secret or an
```

Comment:
The bit about secret, shared, and asymmetric keys is understood by us (since we are in the weeds); but, it might be confusing to others. Technically, `JwtAuthentication` isn't concerned about the type of key, so it should be safe to omit these details.